src/nvim/normal.c

```diff
@@ -7490,9 +7490,9 @@ static void nv_put_opt(cmdarg_T *cap, bool fix_indent)
       // do_put(), which requires the visual selection to still be active.
       if (!VIsual_active || VIsual_mode == 'V' || regname != '.') {
         // Now delete the selected text. Avoid messages here.
+        cap->oap->regname = cap->cmdchar == 'P' ? '_' : NULL;
```

Comment:
`regname` is not a pointer so it should be `NUL`.
```suggestion
        cap->oap->regname = cap->cmdchar == 'P' ? '_' : NUL;
```

Reply:
Oh, it is my fault.